apps/aws-amplify/src/locations/Sidebar.tsx

```diff
@@ -1,17 +1,81 @@
-import React from 'react';
-import { Paragraph } from '@contentful/f36-components';
+import { Paragraph, Button } from '@contentful/f36-components';
 import { SidebarAppSDK } from '@contentful/app-sdk';
 import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+import Axios, { AxiosError } from 'axios';
+import { useState } from 'react';
+
+// use cors-anywhere for local development to test webhook
+(function () {
+  if (process.env.NODE_ENV === 'development') {
+    var cors_api_host = 'cors-anywhere.herokuapp.com';
+    var cors_api_url = 'https://' + cors_api_host + '/';
+    var slice = [].slice;
+    var origin = window.location.protocol + '//' + window.location.host;
+    var open = XMLHttpRequest.prototype.open;
+    XMLHttpRequest.prototype.open = function () {
+      var args: any = slice.call(arguments);
+      var targetOrigin = /^https?:\/\/([^\/]+)/i.exec(args[1]);
+      if (
+        targetOrigin &&
+        targetOrigin[0].toLowerCase() !== origin &&
+        targetOrigin[1] !== cors_api_host
+      ) {
+        args[1] = cors_api_url + args[1];
+      }
+      return open.apply(this, args);
+    };
+  }
+})();
 
 const Sidebar = () => {
   const sdk = useSDK<SidebarAppSDK>();
+  const errorText = 'The AWS website build was not triggered. Try again later.';
+  const [isLoading, setIsLoading] = useState(false);
   /*
      To use the cma, inject it as follows.
      If it is not needed, you can remove the next line.
   */
   // const cma = useCMA();
 
-  return <Paragraph>Hello Sidebar Component (AppId: {sdk.ids.app})</Paragraph>;
+  const buildAmplifyApp = () => {
+    setIsLoading(true);
+    const webhookURL =
+      'https://webhooks.amplify.us-east-1.amazonaws.com/prod/webhooks?id=7a31abcf-724e-4072-9329-8541f4e3bb02&token=Lba6wO2uyDKgDA6BmHcqUqXefzVZa9GG8r23HNQVuc';
```

Comment:
An improvement here would be to place in this a .env file.

Even better yet, just have it call it from the parameters from the Config file using the Sidebar SDK - Matt will know how to do this if you ask him